src/index.html

```diff
@@ -6,8 +6,35 @@
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <title>Product cards</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
 </head>
 <body>
   <h1>Product cards</h1>
+  <div class="container" data-qa="card">
+    <img src="images/imac.jpeg"
+    alt="Apple iMac foto"
+    class="product_foto">
```

Comment:
некорректные переносы
переносятся все атрибуты + отступы добавляются
```suggestion
    <img
      src="images/imac.jpeg"
      alt="Apple iMac foto"
      class="product_foto"
    >
```